src/symbol.rs

```diff
@@ -0,0 +1,152 @@
+use std::borrow::Borrow;
```

Comment:
🗺️ Except where otherwise noted, the contents of this file were cut+pasted from `src/symbol_table.rs`.